cmd/nirvana/project/init.go

```diff
@@ -95,6 +101,7 @@ func (o *initOptions) Install(flags *pflag.FlagSet) {
 	flags.StringVar(&o.ImageSuffix, "image-suffix", "", "Docker image suffix")
 	flags.StringVar(&o.BuildImage, "build-image", "golang:latest", "Golang image for building the project")
 	flags.StringVar(&o.RuntimeImage, "runtime-image", "debian:jessie", "Docker base image for running the project")
+	flags.StringVar(&o.Pkgmgr, "pkgmgr", "dep", "Package manager of project, dep or mod")
```

Comment:
The `--pkgmgr` is not like other flags. `--package-manager` is better.

Reply:
ok

Reply:
I have modified it